modules/op-tkn-pipeline-management.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/tkn_cli/op-tkn-references.adoc
+
+[id="op-tkn-pipeline-management_{context}"]
+= Pipelines management commands
+
+== tkn pipeline
+Manage pipelines.
+
+.Example: Display help
+----
+$ pipeline --help
+----
+
+== tkn pipeline create
+Create a pipeline in a namespace.
+
+.Example: Create a pipeline defined by the `mypipeline.yaml` file in a namespace
+-----
+$ tkn pipeline create -f mypipeline.yaml -n myspace
```

Comment:
There is inconsistent style... sometimes you specify `-n myspace` and sometimes not, I suggest to unify. Not specifying it at all means use current namespace which IMHO is what users do 99% of time.

Reply:
Hi Pavol, as discussed I have removed " in a  namespace" from the cmd description.